editions/tw5.com/tiddlers/system/filter-run-template.tid

```diff
@@ -0,0 +1,50 @@
+code-body: yes
+created: 20230316112235083
+list-before: $:/core/ui/ViewTemplate/body
+modified: 20230318144728713
+tags: $:/tags/ViewTemplate
+title: $:/editions/tw5.com/filter-run-template
+type: text/vnd.tiddlywiki
+
+\define .op-place()
+<$macrocall $name=".if"
+	cond="""$(op-name)$"""
+	then="<<.place '$(op-name)$'>> = "
+	else=""/>
+\end
+
+\define .op-row()
+<$macrocall $name=".if"
+	cond="""$(op-body)$"""
+	then="""<tr><th align="left">$(op-head)$</th><td><<.op-place>>$(op-body)$</td></tr>"""
+	else=""/>
+\end
+
+\whitespace trim
+
+<$list filter="[all[current]tag[Named Filter Run Prefix]]">
+	<p>Parents: <$macrocall $name=".recursiveParentTag" current=<<currentTiddler>> stop="Filters"/></p>
+	<$let op-head="" op-body="" op-name="">
+		<$list filter="[all[current]has[from-version]]" variable="listItem">
+			<$macrocall $name=".from-version" version={{!!from-version}}/>
+		</$list>
```

Comment:
Yes. But filter run prefixes are not operators. So I think they can be visually different. It only needs to be consistent within Filter Run Prefixes. And imo it does look better that way. You can test it yourself in the preview. If you change it to be shown below the table it does not look good.

<details><summary>Click to show image</summary>

![image](https://user-images.githubusercontent.com/374655/226482812-75281165-be73-495b-9073-d48706d528fc.png)

</details>

Reply:
> It only needs to be consistent within Filter Run Prefixes.

@pmario: I don't think so, since it's not specific to them. Whenever there is an overview table and a "new from" label in the same tiddler, their order should be consistent.

Reply:
I moved it below the table